sha256-hasher/src/lib.rs

```diff
@@ -1,16 +1,24 @@
 #![no_std]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
-#[cfg(all(feature = "sha2", not(target_os = "solana")))]
-use sha2::{Digest, Sha256};
+
 use solana_hash::Hash;
+#[cfg(any(target_os = "solana", target_arch = "bpf"))]
+pub use {
+    core::mem::MaybeUninit, solana_define_syscall::definitions::sol_sha256, solana_hash::HASH_BYTES,
+};
```

Comment:
This shouldn't be `pub use`, right?
```suggestion
use {
    core::mem::MaybeUninit, solana_define_syscall::definitions::sol_sha256, solana_hash::HASH_BYTES,
};
```